CRAYSAT-1882: sat bootsys shutdown and boot improvements doc changes (#5262)
Cray-HPE/docs-csm: operations/power_management/Power_On_and_Start_the_Management_Kubernetes_Cluster.md

   Are the above NCN groupings and exclusions correct? [yes,no] yes
   INFO: Starting console logging on ncn-s003,ncn-s001,ncn-w002,ncn-m003,ncn-w004,ncn-m002,ncn-s002,ncn-w001,ncn-w003.
   Powering on NCNs and waiting up to 900 seconds for them to be reachable via SSH: ncn-m002, ncn-m003
   INFO: Sending IPMI power on command to host ncn-s002
   INFO: Sending IPMI power on command to host ncn-s001
   INFO: Sending IPMI power on command to host ncn-s003
   Waiting for condition "Hosts accessible via SSH" timed out after 300 seconds
   ERROR: Unable to reach the following NCNs via SSH after powering them on: ncn-m003, ncn-s002.. Troubleshoot the issue and then try again.
   ```

   In the preceding example, the `ssh` command to the NCN nodes timed out and reported `ERROR` messages. Repeat the above step until you see `Succeeded with boot of other management NCNs.` Each iteration should get further in the process.

>    NOTE: During the Power on if the ceph does not become healthy it would timeout after which, You would have a prompt as shown below to proceed further or exit to fix the ceph health.

Reviewer: ```suggestion
   NOTE: During power on, if Ceph does not become healthy, it will time out, prompting to either proceed further or exit to fix Ceph health.
```

   If 'yes' is given as the input, it would skip to check the ceph status and proceed further.

   Example output:

   ```text
    INFO: Checking Ceph health
    Waiting for condition "Ceph cluster in healthy state" timed out after 60 seconds
    ERROR: Failed to unfreeze Ceph on storage NCNs: Ceph is not healthy. Please correct Ceph health and try again.
    eph is not healthy. Do you want to continue anyway? [yes,no] yes
    INFO: Continuing despite Ceph not being healthy as per user's input, make sure to verify it later.
    INFO: Checking whether ceph filesystem is mounted on /etc/cray/upgrade/csm.
    ```

>    If 'No' is given as the input, it would exit the execution, you can fix the ceph health, See [Manage Ceph Services](../utility_storage/Manage_Ceph_Services.md) for Ceph troubleshooting

Reviewer: ```suggestion
   If 'No' is given as the input, it would exit the execution. To fix the Ceph health, see [Manage Ceph Services](../utility_storage/Manage_Ceph_Services.md) for Ceph troubleshooting.
```
